src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -1320,14 +1292,14 @@ def get_age_and_sex(measure_str):
     age_end_dict = {"early_neonatal": 0.01917808, "late_neonatal": 0.07671233}
     df["age_start"] = df["age"].replace(age_start_dict)
     df["age_end"] = df["age"].replace(age_end_dict)
-    df["year_start"] = 2019
-    df["year_end"] = 2020
+    df["year_start"] = 2021
+    df["year_end"] = 2022
     df = df.drop("age", axis=1)
 
-    new_row_1 = [0] * 1000 + ["Female", 0.07671233, 1.0, 2019, 2020]
-    new_row_2 = [0] * 1000 + ["Male", 0.07671233, 1.0, 2019, 2020]
-    new_row_3 = [0] * 1000 + ["Female", 1.0, 5.0, 2019, 2020]
-    new_row_4 = [0] * 1000 + ["Male", 1.0, 5.0, 2019, 2020]
+    new_row_1 = [0] * 500 + ["Female", 0.07671233, 1.0, 2021, 2022]
```

Comment:
In all the places where you have changed 1000 to 500, you should use the variable from metadata for draw count. That way you don't have to replace every single one of these and you can just change the actual value in the variable. You should do this everywhere (except the model spec)

Reply:
Replaced 500 with metadata.DRAW_COUNT